Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -454,6 +454,7 @@ - (BOOL)prefersStatusBarHidden
         interface->invalidate();
         [model->layerHostView() removeFromSuperview];
         model->setLayerHostView(nullptr);
+        model->playerLayer().fullscreenModel = nullptr;
```

Comment:
it think it would have been nicer to keep the same style as before:
`[model->playerLayer() setFullscreenModel:nullptr];`